localenv/happy-life-bank/docker-compose.yml

```diff
@@ -25,8 +25,10 @@ services:
       - ../happy-life-bank/seed.yml:/workspace/seed.yml
       - ../happy-life-bank/private-key.pem:/workspace/private-key.pem
     depends_on:
-      - cloud-nine-mock-ase
-      - happy-life-backend
+      happy-life-backend:
+        condition: service_healthy
+      cloud-nine-mock-ase:
+        condition: service_started
```

Comment:
I wonder why we had the `cloud-nine-mock-ase` dependency originally, do we need it?

Reply:
This was introduced in https://github.com/interledger/rafiki/pull/1812. All `happy-life` services depend on the `cloud-nine` ones to avoid building the same images twice. If we remove the `depends_on`, on a fresh localenv startup all the images will be built twice since they are not found locally.

Reply:
Got it, thanks @raducristianpopa !